models/dashboards/pevr/pbi_tables/pevr_indicateur_des.sql

```diff
@@ -1,119 +1,119 @@
-{#
-Dashboards Store - Helping students, one dashboard at a time.
-Copyright (C) 2023  Sciance Inc.
-
-This program is free software: you can redistribute it and/or modify
-it under the terms of the GNU Affero General Public License as
-published by the Free Software Foundation, either version 3 of the
-License, or any later version.
-
-This program is distributed in the hope that it will be useful,
-but WITHOUT ANY WARRANTY; without even the implied warranty of
-MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-GNU Affero General Public License for more details.
-
-You should have received a copy of the GNU Affero General Public License
-along with this program.  If not, see <https://www.gnu.org/licenses/>.
-#}
-{{ config(alias="indicateur_des") }}
-
-
-with
-    src as (
-        select
-            sch.annee,
-            sch.annee_scolaire,
-            src.fiche,
-            sch.school_friendly_name,
-            case
-                when mentions.ind_reus_sanct_charl = 'O' then 1.0 else 0.0
-            end as 'ind_obtention'
-        from {{ ref("stg_perimetre_eleve_diplomation_des") }} as src
-        left join {{ ref("i_e_ri_mentions") }} as mentions on src.fiche = mentions.fiche
-        inner join {{ ref("dim_mapper_schools") }} as sch on src.id_eco = sch.id_eco
-        where
-            mentions.prog_charl = '6200'
-            and sch.annee
```

Comment:
@busquef prog_charl = 6200 exclu les élèves qui vont s'inscrir aux adultes pour finaliser des unités manquantes. tu peux testet avec regime_sanct_charl in ('A3','J5')

Reply:
ca na rien à voir avec 7 ans ou 6 ans. ton point de départ est les élèves inscrit à une année donnée

Reply:
hmm effectivement

Je regarde ça.

Reply:
J'ai un cas que c'est du GENA12 et le type de diplome est TDG. je ne connais pas ce type de diplome

Reply:
![image](https://github.com/user-attachments/assets/b2eee697-bd91-4ca3-a0bc-3beb608eb824)

Reply:
il te faut un seqid (partition by fiche, annee (eventuellement mois ou session) order by date_exec_sanct desc)

Reply:
C'est ajouté.